src/domain/users/index.ts

```diff
@@ -15,6 +17,11 @@ import {
 // for a more precise "regex"
 const PhoneNumberRegex = /^\+\d{7,14}$/i // FIXME {7,14} to be refined
 
+const KratosUserIdRegex =
+  /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i
+
+const EmailAddressRegex = /^[\w-.+]+@([\w-]+\.)+[\w-]{2,4}$/i
```

Comment:
this regex is giving a `pattern error`, it works if I remove `-` form `[\w-.+]`. could you please verify?

Reply:
Where are you getting that error?